tests/Doctrine/Tests/Models/EagerBatched/Article.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: wouter
+ * Date: 11/18/15
+ * Time: 1:00 PM
+ */
+
+namespace Doctrine\Tests\Models\EagerBatched;
+
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping\Table;
+use Doctrine\Tests\Models\EagerBatched\Tag;
+
+/**
+ * Class Article
+ * @package Doctrine\Tests\Models\EagerBatched
+ *
+ * @Entity()
+ * @Table(name="eager_batched_article")
+ */
+class Article
+{
+
+    /**
+     * @Id @Column(type="integer", name="id") @GeneratedValue
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * @Column(type="string")
+     * @var string|null
+     */
+    protected $title;
+
+    /**
+     * @OneToMany(targetEntity="Tag",mappedBy="article",cascade={"persist"},fetch="EAGER_BATCHED")
+     * @var Tag[]|Collection
+     */
+    protected $tags;
```

Comment:
`$tags` needs to be initialized in the constructor to a new instance of the `Doctrine\Common\Collections\ArrayCollection` class. See [doctrine documentation](http://doctrine-orm.readthedocs.org/projects/doctrine-orm/en/latest/reference/association-mapping.html#one-to-many-bidirectional) for details